ui/timeline/src/commonMain/kotlin/social/androiddev/timeline/FeedCard.kt

```diff
@@ -0,0 +1,217 @@
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import social.androiddev.common.composables.UserAvatar
+import social.androiddev.common.theme.MastodonTheme
+
+/**
+ * Stateful
+ */
+@Composable
+fun FeedCard(
+    state: FeedItemState,
+    contentPadding: PaddingValues = PaddingValues(8.dp),
+    modifier: Modifier = Modifier,
+    backgroundColor: Color = MaterialTheme.colors.surface,
+    contentColor: Color = contentColorFor(backgroundColor),
+) {
+    FeedCard(
+        userAvatarUrl = state.userAvatarUrl,
+        date = state.date,
+        username = state.username,
+        userAddress = state.acctAddress,
+        toot = state.toot,
+        contentPadding = contentPadding,
+        modifier = modifier,
+        backgroundColor = backgroundColor,
+        contentColor = contentColor
+    )
+}
+
+/**
+ * Stateless
+ */
+@Composable
+fun FeedCard(
+    userAvatarUrl: String,
+    date: String,
+    username: String,
+    userAddress: String,
+    toot: String?,
+    contentPadding: PaddingValues = PaddingValues(8.dp),
+    modifier: Modifier = Modifier,
+    backgroundColor: Color = MaterialTheme.colors.surface,
+    contentColor: Color = contentColorFor(backgroundColor),
+) {
+    Card(
+        modifier = modifier,
+        backgroundColor = backgroundColor,
+        contentColor = contentColor,
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight()
+                .padding(contentPadding),
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            UserAvatar(
+                modifier = Modifier
+                    .size(72.dp)
+                    .clip(CircleShape)
+                    // TODO We should add support for a default image placeholder
+                    // in the case of loading or error
+                    .background(Color.Red),
```

Comment:
this should be done inside the UserAvatar
Currently a placeholder is used. But I would change it to a composable param instead.

Reply:
Good to know -  I missed that we had that. Will Update next commit